Chummer/Backend/Uniques/Spell.cs

```diff
@@ -77,7 +78,7 @@ public Spell(Character objCharacter)
         /// <param name="blnExtended">Whether or not the Spell should be marked as Extended.</param>
         /// <param name="blnAlchemical">Whether or not the Spell is one for an alchemical preparation.</param>
         /// <param name="objSource">Enum representing the actual type of spell this object represents. Used for initiation benefits that would grant spells.</param>
-        public void Create(XmlNode objXmlSpellNode, string strForcedValue = "", bool blnLimited = false, bool blnExtended = false, bool blnAlchemical = false, Improvement.ImprovementSource objSource = Improvement.ImprovementSource.Spell)
+        public virtual void Create(XmlNode objXmlSpellNode, string strForcedValue = "", bool blnLimited = false, bool blnExtended = false, bool blnAlchemical = false, Improvement.ImprovementSource objSource = Improvement.ImprovementSource.Spell)
```

Comment:
It doesn't seem like you need the `Create` method to be `virtual`?

Reply:
Good catch, missed that one. Before I implemented `CreateByMapping()`, I had a `override Create()` in the derived classes.